src/Resources/skeleton/doctrine/Entity.tpl.php

```diff
@@ -2,13 +2,15 @@
 
 namespace <?= $namespace ?>;
 
-<?php if ($api_resource): ?>use ApiPlatform\Core\Annotation\ApiResource;
+<?php if ($api_resource && class_exists(\ApiPlatform\Metadata\ApiResource::class)): ?>use ApiPlatform\Metadata\ApiResource;
+<?php elseif ($api_resource): ?>use ApiPlatform\Core\Annotation\ApiResource;
```

Comment:
is it related to this PR?

Reply:
Yea, this is from #968 - what's it doing in here?

Reply:
done - think this made it in with a merge maybe? reverted